### ISSUE TITLE
The valhook setter causes other plugins to fail

### ISSUE THREAD

**Issue by RobinHerbots** [customd/jquery-number#71]
In the valhook setter code you don't always call the original valhook to set the value.  This can cause unexpected behavior in others plugins which also use the valhook.

```
           // Otherwise, don't worry about other valhooks, just run ours.
            return el.value = num;
        }
    };
```

**Comment by samatcd** [comment]
What would you expect to happen here? When running other valHook plugins, what value should we return?

Do you have an example / use case you're trying to achieve?

**Comment by RobinHerbots** [comment]
@samatcd , 

For example in the setter.  You do when yuo don't find $this.data('numFormat'); you call the origHookSet.  When you find it you process the value and set it with the above line.  It should also call the origHookSet (see comment in the code example

Something like this: 

```
$.valHooks.text.set = function( el, val )
	{
		// Get the element, and its data.
		var $this	= $(el),
			data	= $this.data('numFormat');

		// Does this element have our data field?
		if( !data )
		{

			// Check if the valhook function already exists
			if( $.isFunction( origHookSet ) )
			{
				// There was, so go ahead and call it
				return origHookSet(el,val);
			}
			else
			{
				// No previous function, return undefined to have jQuery
				// take care of retrieving the value
				return undefined;
			}
		}
		else
		{
			var num = $.number( val, data.decimals, data.dec_point, data.thousands_sep );

			// Make sure empties are set with correct signs.
//			if(val.indexOf('-') === 0 && +num === 0)
//			{
//				num = '-'+num;
//			}
	                //cal the original valhook setter with your processed value
                        // this way the original behavior if any of the original valhooksetter is also applied
			return origHookSet(el,num);
		}
	};
```

**Comment by samatcd** [comment]
Ahh, Ok, that makes sense. If you submit that as a pull request, I'll merge it in.

**Comment by samatcd** [comment]
Fixed as per merge #72
